lib/Models/FunctionParameter.js

```diff
@@ -204,8 +204,22 @@ FunctionParameter.prototype.load = function() {
     return this._loadingPromise;
 };
 
+/**
+ * Represents value as string.
+ * @param {Object} value Value to format as string.
+ * @return {String} String representing value.
+ */
 FunctionParameter.prototype.formatValueAsString = function(value) {
     return defined(value) ? value.toString() : '-';
 };
 
+/**
+ * Returns a GeoJSON feature that represents value, if it makes sense to do so.
+ * @param {Object} value Value to get GeoJSON feature for.
+ * @return {Object} JSON feature or undefined.
+ */
+FunctionParameter.prototype.getGeoJsonFeature = function(value) {
```

Comment:
I'm not a huge fan of having this here.  It'd be better to just make it an optional method.  You could introduce an intermediate base class, like `GeometryFunctionParameter` if you want.